base/workflow/R/create_execute_test_xml.R

```diff
@@ -18,9 +18,14 @@
 #'   analysis. Default = `"NPP"`
 #' @param sensitivity (logical) Whether or not to perform a sensitivity analysis
 #'   (default = `FALSE`)
-#' @param db_bety_username (character) BETY username for workflow (default = `"bety"`)
-#' @param db_bety_password (character) BETY password for workflow (default = `"bety"`)
-#' @param db_bety_hostname (character) BETY hostname for workflow (default = `"localhost"`)
+#' @param db_bety_username (character) BETY username for workflow. Default is
+#'   the value from the `config.php`.
+#' @param db_bety_password (character) BETY password for workflow. Default is
+#'   the value from `config.php`.
+#' @param db_bety_hostname (character) BETY hostname for workflow. Default is
+#'   the value from `config.php`
+#' @param db_bety_hostname (character) BETY connection port for workflow.
+#'   Default is the value from `config.php`
```

Comment:
Would be good to specify that config.php is read from `[pecan_path]/web/config.php` -- from these param descriptions I'd probably have guessed `[pecan_path]/config.php`. This could probably go in a `@details` section rather than repeating for all the `db_bety*` descriptions.